docs/source/technical_tutorials/search/collections.ipynb

```diff
@@ -1,18 +1,41 @@
 {
```

Comment:
<p>Maybe emphasize that these are CMR collections we're searching, more aligned with the title of the tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>